models/standalone-price/src/transformers.ts

```diff
@@ -97,18 +87,27 @@ const transformers = {
       'channel',
       'tiers',
     ],
-    addFields: ({ fields }) => {
+    replaceFields: ({ fields }) => {
```

Comment:
`replaceFields` correctly retrieves the `fields` values.
 
When `addFields` is invoked, within the body of that function, every property of the `fields` object is interpreted as an anonymous function, for some reason.

Therefore, the previous implementation of the `addField` function was irrelevant because it relied on precise property values (`fields.customerGroup.id `and `fields.channel.id)`, which were instead evaluated as anonymous functions, causing a mismatch of id's and key's in refs values.